src/features/events/components/EventWarningIcons.tsx

```diff
@@ -32,7 +31,7 @@ const EventWarningIcons: FC<EventWarningIconsProps> = ({ compact, model }) => {
       numRemindersSent={
         participants.data?.filter((p) => !!p.reminder_sent).length ?? 0
       }
-      numSignups={data.num_participants_available}
+      numSignups={model.getPendingSignUps().length ?? 0}
```

Comment:
Same comment here as in #1358:

The `??` does nothing here, I don't think. According to the types, `model.getPendingSignUps()` always returns an array, and there are no optional chaining operators (`?`), so the nullish coalescing operator (`??`) has no effect. It should be possible to just do:

```suggestion
      numSignups={model.getPendingSignUps().length}
```

Reply:
True, I will make the changes asap! Thanks!